tests/test_cli.py

```diff
@@ -11,13 +10,27 @@
 from conftest import FIXTURES_FOLDER
 
 
-@pytest.fixture()
-def mock_app_run(mock_get_function_from_name):
-    test_server = MagicMock(wraps=connexion.FlaskApp(__name__))
-    test_server.run = MagicMock(return_value=True)
-    test_app = MagicMock(return_value=test_server)
-    mock_get_function_from_name.return_value = test_app
-    return test_app
+@pytest.fixture(scope="function")
+def mock_app_run(app_class, monkeypatch):
```

Comment:
`run` now needs to be mocked differently, since it needs to call `_build_middleware_stack`. Also extended this to run for both the `FlaskApp` and `AsyncApp`.